azure-pipelines.yml

```diff
@@ -0,0 +1,85 @@
+trigger:
+  branches:
+    include:
+      - '*'
+
+pr: none
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+resources:
+  repositories:
+    - repository: platform-libraries-input-mapping
+      type: github
+      endpoint: keboola
+      name: keboola/platform-libraries-input-mapping
+
+variables:
+  DOCKER_BUILDKIT: 1
+  COMPOSE_DOCKER_CLI_BUILD: 1
+
+stages:
+  - stage: build
+    displayName: Build
+    dependsOn: []
+    jobs:
+      - job: checkChanges
+        displayName: 'Check changes'
+        steps:
+          - script: |
+              ./bin/ci-find-changes.sh \
+                input-mapping:libs/input-mapping
```

Comment:
`inputMapping:libs/input-mapping`? po vzoru `job-queue` :wink:

Reply:
I did that on purpose :-D

Reply:
I see, but than you have weird stuff like `changedProjects_input-mapping`, isn't `changedProjects_inputMapping` much nicer? :wink:

Reply:
ok, I'll switch it. I was trying to keep it consistent, but you're right that having dP and _ and - in a single name is almost violence 😄

Reply:
will you? :slightly_smiling_face:

Reply:
I did :-)

Reply:
sorry, I missed the update :facepalm: